client/src/views/Register.tsx

```diff
@@ -1,7 +1,71 @@
+import React, { useState } from "react";
+import { useAuthStore } from "../store/authStore";
+import InputField from "../components/InputField";
+import FormError from "../components/FormError";
+
 export default function Register() {
```

Comment:
this route you need to have access as a user when you already login.

Reply:
I think he means that a user should not have access to the register route if they are already logged in